docs/accessing-instances.md

```diff
@@ -0,0 +1,83 @@
+# Accessing cluster instances
+
+## Overview
+
+After running `clusterctl` the new cluster will be deployed. This document
+explains how to access it's nodes through SSH.
+
+## Prerequisites
+
+* `clusterctl` was successfully executed and the cluster is up and running in
+an Azure environment
+* the `cluster-api-provider-azure.sigs.k8s.io` has been created in Azure and saved
+as `$HOME/.ssh/cluster-api-provider-azure`
+
+## Azure architecture
+
+After cluster creation none of the cluster instances are exposed to the
+internet, i.e. cannot be accessed. To make it accessible, `clusterctl` also
+creates a bastion node
+
+### bastion node
+
+bastion node is created in a public subnet and provides SSH access from the
+world. It runs the official Ubuntu 18.04 Linux image.
+
+### cluster nodes
+
+cluster nodes are either control plane or worker nodes. They all run the
+official Ubuntu 18.04 Linux image and are deployed in a private subnet.
+
+## Accessing cluster nodes
+
+Cluster nodes should be accessed through the bastion node that is created
+along with the cluster. `cluster-api-provider-azure.sigs.k8s.io` SSH key
+should be used for authentication.
+
+### Setting up the SSH key path
+
+> Assummig the `cluster-api-provider-azure.sigs.k8s.io` is stored in
+`$HOME/.ssh/cluster-api-provider-azure`
+
+```bash
+export CLUSTER_SSH_KEY=$HOME/.ssh/cluster-api-provider-azure
+```
+
+### Obtain public IP address of the bastion node
+
+> Your credentials must let you query EC2 API
+
+```bash
+export BASTION_HOST=$(azure ec2 describe-instances --filter='Name=tag:Name,Values=<CLUSTER_NAME>-bastion' \
```

Comment:
I see that this has a lot of amazon specific stuff. I understand that the intent is to fix these in a follow-up PR. However, from past experiences, it's much harder to track all of the doc debt after merge. Is it possible to scope this in another PR?

Reply:
We can exclude these docs which have a lot of aws content basically.

Reply:
Meant to remove that file. Thanks for catching. Just fixed!